clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/actions/ReconnectAction.kt

```diff
@@ -0,0 +1,17 @@
+package org.utbot.cpp.clion.plugin.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.components.service
+import org.utbot.cpp.clion.plugin.UTBot
+import org.utbot.cpp.clion.plugin.client.ClientManager
+
+class ReconnectAction: AnAction(UTBot.message("actions.reconnect")) {
+    override fun actionPerformed(e: AnActionEvent) {
+        e.project!!.service<ClientManager>().restartClient()
+    }
+
+    override fun update(e: AnActionEvent) {
+        e.presentation.isEnabledAndVisible = e.project != null
```

Comment:
Do we need to extend this availability by ON/OFF boolean from per-project UTBotCpp plugin option?

Reply:
I didn't add ON/OFF functionality for plugin yet. So no